spec/support/capybara.rb

```diff
@@ -41,7 +41,7 @@
 
 Capybara.server = :puma, { Silent: true }
 
-Capybara.default_max_wait_time = (ENV['CAPYBARA_WAIT_TIME_SECONDS'] || '0.5').to_f
+Capybara.default_max_wait_time = 0
```

Comment:
Why remove the capacity to set a longer wait time locally with an environment variable? I'm doing a full feature test run locally on this branch, and I saw some failures go by on document capture, which already gives the user a lot of feedback about the process taking a while.

Just finished! (45 minutes altogether) Failed examples:

rspec ./spec/features/accessibility/idv_pages_spec.rb:65 # Accessibility on IDV pages IDV pages doc auth steps accessibility on mobile
rspec './spec/features/idv/steps/phone_step_spec.rb[1:10:2:2:1]' # idv phone step after the max number of attempts behaves like verification step max attempts after completing one less than the max attempts allows the user to continue if their last attempt is successful
rspec ./spec/features/idv/steps/gpo_step_spec.rb:92 # idv gpo step Verified resets password, requests GPO, then signs in using SP shows the user the GPO code entry screen
rspec ./spec/features/users/sign_in_spec.rb:260 # Sign in session approaches timeout user sees warning before session times out
rspec ./spec/features/users/sign_in_spec.rb:279 # Sign in session approaches timeout user can continue browsing
rspec ./spec/features/idv/in_person_spec.rb:119 # In Person Proofing works for a happy path
rspec ./spec/features/idv/in_person_spec.rb:245 # In Person Proofing allows the user to cancel and start over from the beginning
rspec ./spec/features/idv/in_person_spec.rb:258 # In Person Proofing allows the user to go back to document capture from prepare step
rspec ./spec/features/idv/in_person_spec.rb:406 # In Person Proofing verify address by mail (GPO letter) lets the user clear and start over from gpo confirmation
rspec ./spec/features/idv/in_person_spec.rb:376 # In Person Proofing verify address by mail (GPO letter) requires address verification before showing instructions
rspec ./spec/features/idv/in_person_spec.rb:426 # In Person Proofing transliteration shows validation errors
rspec ./spec/features/idv/in_person_spec.rb:501 # In Person Proofing validate_id_and_residential_addresses feature flag enabled captures the address, address line 2, city, state and zip code
rspec './spec/features/idv/in_person_spec.rb[1:5:1:1]' # In Person Proofing after in-person proofing is completed and passed for a partner using oidc sends a survey when they share information with that partner
rspec './spec/features/idv/in_person_spec.rb[1:5:2:1]' # In Person Proofing after in-person proofing is completed and passed for a partner using saml sends a survey when they share information with that partner
rspec ./spec/features/idv/in_person_spec.rb:23 # In Person Proofing ThreatMetrix review pending allows the user to continue down the happy path
rspec ./spec/features/idv/in_person_spec.rb:325 # In Person Proofing with hybrid document capture resumes desktop session with in-person proofing
rspec ./spec/features/idv/doc_capture/document_capture_step_spec.rb:75 # doc capture document capture step does not advance original session with errors
rspec ./spec/features/idv/doc_capture/document_capture_step_spec.rb:61 # doc capture document capture step advances original session once complete
rspec ./spec/features/idv/doc_capture/document_capture_step_spec.rb:108 # doc capture document capture step when using async uploads advances original session once complete
rspec ./spec/features/idv/doc_capture/document_capture_step_spec.rb:125 # doc capture document capture step when using async uploads does not advance original session with errors
rspec ./spec/features/idv/doc_capture/document_capture_step_spec.rb:149 # doc capture document capture step when using async uploads with attention with barcode result advances original session only after confirmed
rspec ./spec/features/idv/doc_capture/document_capture_step_spec.rb:91 # doc capture document capture step with attention with barcode result advances original session only after confirmed
rspec ./spec/features/two_factor_authentication/sign_in_spec.rb:63 # Two Factor Authentication When the user has not set up 2FA with international phone that does not support voice delivery updates international code as user types

Reply:
Can you provide more detail about those errors? I had initially encountered some failures in my environment because of configuration in my local `config/application.yml` was taking precedent, but they went away after resetting my configuration, and weren't related to wait timeouts.

The description of the pull request explains why I think the default wait is an anti-pattern. Allowing this to be configurable would imply that there are instances where we want to tolerate the application to be unresponsive in areas that we aren't explicitly expecting it to be unresponsive. Setting this to a single value also avoids inconsistencies like where previously CI was more lax than local development, which contributed to flakiness of tests, difficulty in reproducing them, and lack of clarity that a distinction existed at all between CI and local development.

Reply:
Emailed error output as far back as my terminal history goes.

I understand the reasoning behind changing it to 0 on CI/CD. And I think we need to be able to run feature tests locally.

Reply:
I would suggest changing the default from `0.5` to `0` without hard-coding a magic number replacing the variable. Those of us with inconsistently failing tests still need this knob to twiddle as we attempt to improve the problem.

Reply:
@jskinne3 Could you share some detail about the specs you're seeing failing on this branch? My hope is that these changes address all of the intermittent failings related to this setting and normalize the behavior across systems to prevent the possibility of any "wait"-related intermittent failures from being introduced in the future.

Reply:
Sure. I started a `bundle exec make_test` on this branch. I'll email you the output when finished.

Reply:
@soniaconnolly @jskinne3 I added the environment variable back in ebd4c5e81. Can you please take a look?